includes/debug-info.php

```diff
@@ -0,0 +1,245 @@
+	add_action(
+		'plugins_loaded',
+		function() {
+			add_filter( 'debug_information', __NAMESPACE__ . '\add_debug_info' );
+		}
+	);
+	add_action( 'admin_enqueue_scripts', __NAMESPACE__ . '\enqueue_scripts' );
+}
+
+/**
+ * Enqueue scripts/styles for site health.
+ *
+ * @since  2.0.0
+ *
+ * @param int $hook Hook suffix for the current admin page.
+ */
+function enqueue_scripts( $hook ) {
+	if ( 'site-health.php' !== $hook ) {
+		return;
+	}
+	wp_enqueue_style( 'dt-site-health', plugins_url( '/dist/css/admin-site-health.min.css', __DIR__ ), [], DT_VERSION );
+}
+
+/**
+ * Setup actions and filters
+ *
+ * @since 2.0.0
+ */
+/**
+ * Add distributor debug information to Site Health screen.
+ *
+ * @see WP_Debug_Data::debug_data
+ * @filter debug_information
+ *
+ * @param array $info The full array of site debug information.
+ * @return array Filtered debug information.
+ *
+ * @since 2.4.0
+ */
+function add_debug_info( $info ) {
+
+	$plugin_data = get_plugin_data( WP_PLUGIN_DIR . '/' . DT_PLUGIN_FILE );
+	$text_domain = $plugin_data['TextDomain'];
+	$defaults    = [
+		'email'         => '',
+		'valid_license' => false,
+		'license_key'   => '',
+	];
+
+	$all_settings = wp_parse_args(
+		(array) get_option( 'dt_settings' ),
+		$defaults
+	);
+
+	// Get settings without license data.
+	$settings = array_diff_key( $all_settings, $defaults );
+	$fields   = [
+		[
+			'label' => __( 'Version', 'distributor' ),
+			'value' => $plugin_data['Version'],
+		],
+	];
+
+	if ( false === DT_IS_NETWORK || is_network_admin() ) {
+		$fields[] = [
+			'label' => __( 'Valid registration', 'distributor' ),
+			'value' => $all_settings['valid_license'] ? __( 'Yes', 'distributor' ) : __( 'No', 'distributor' ),
+		];
+		$fields[] = [
+			'label' => __( 'Registration email', 'distributor' ),
+			'value' => $all_settings['email'] ? $all_settings['email'] : __( 'N/A', 'distributor' ),
+		];
+	}
+
+	$fields = array_merge(
+		$fields,
+		[
+			[
+				'label' => __( 'Settings', 'distributor' ),
+				'value' => [
+					__( 'Override Author Byline', 'distributor' ) => $all_settings['override_author_byline'] ? __( 'Yes', 'distributor' ) : __( 'No', 'distributor' ),
+					__( 'Media Handling', 'distributor' ) => 'attached' === $all_settings['media_handling'] ? __( 'Featured image and attached images', 'distributor' ) : __( 'Featured image only', 'distributor' ),
+				],
+			],
+			[
+				'label' => __( 'Internal Connections', 'distributor' ),
+				'value' => get_formatted_internal_connnections(),
+			],
+			[
+				'label' => __( 'External Connections', 'distributor' ),
+				'value' => get_formatted_external_connnections(),
+			],
+		]
+	);
+
+	$info[ $text_domain ] = [
+		'label'  => $plugin_data['Name'],
+		'fields' => $fields,
+	];
+
+	return $info;
+}
+
+/**
+ * Get and format internal connections.
+ *
+ * @return array
+ */
+function get_formatted_internal_connnections() {
+	if ( empty( \Distributor\Connections::factory()->get_registered()['networkblog'] ) ) {
```

Comment:
Not a blocker, but I see PHP `use` being used elsewhere in the plugin. Why not use it in this file?

Reply:
I don't think I get your idea :D, do you mind explaining that again 😄?

Reply:
For example, you could put `use Distributor\Connections;` at the top of this file, and make lines like this a little less cluttered:
```PHP
<?php

use Distributor\Connections;

... 

if ( empty( Connections::factory()->get_registered()['networkblog'] ) ) {
```

Reply:
Oh, I see now, what a silly question 😳! Updated. Thank you!